libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -19,6 +19,12 @@ using namespace Runtime;
 
 namespace eosio { namespace chain {
 
+   struct account_module {
+      digest_type digest;
+      std::shared_ptr<wasm_instantiated_module_interface> module_interface;
```

Comment:
This doesn't seem to be used anymore, meaning the `map<account_name, account_module> account_module_map` can probably become a `map<account_name, digest_type>`

That would remove the need to change the `std::unique_ptr` to a `std::shared_ptr` as well which is nicer.